### PR TITLE
Fix app crashes after autofocus of LNURL amount input. Define bugsnag component outside of app component to avoid subtree's DOM nodes and states beeing destroyed

### PR DESCRIPTION
**Pull request by KKA11010**

### CODE REVIEW

File: src/components/pages/Lightning/payInvoice.tsx [hunk 1]
```diff
@@ -76,7 +75,7 @@ export default function PayInvoicePage({ navigation, route }: TPayLNInvoicePageP
 				})
 			} catch (e) {
 				l(e)
-				openPrompt(e instanceof Error ? e.message : 'An error occured while paying the invoice.')
+				openPromptAutoClose(false, e instanceof Error ? e.message : 'An error occured while paying the invoice.')
```
Comment (BilligsterUser) [inline src/components/pages/Lightning/payInvoice.tsx:78]: use isErr

**Review by BilligsterUser — CHANGES_REQUESTED** [review]

File: src/wallet/index.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ import {
 	getMintBalance, getMintsUrls
 } from '@db'
 import { l } from '@log'
-import { isCashuToken } from '@src/util'
+import { isCashuToken, isErr } from '@src/util'
```
Comment (BilligsterUser) [inline src/wallet/index.ts:12]: unused?

**Review by BilligsterUser — CHANGES_REQUESTED** [review]

File: config/app.config.ts [hunk 3]
```diff
@@ -74,7 +74,7 @@ const config: ExpoConfig = {
 			{
 				cameraPermission: 'Allow eNuts to access camera.'
 			}
-		]
+		],'./android-manifest.plugin.js'
```
Comment (BilligsterUser) [inline config/app.config.ts:77]: Pls Remove

**Review by BilligsterUser — CHANGES_REQUESTED** [review]

File: src/wallet/index.ts [hunk 4]
```diff
@@ -57,30 +57,23 @@ export function getMintInfo(mintUrl: string) {
...
-				// eslint-disable-next-line no-await-in-loop
-				const w = await getWallet(t.mint)
-				// eslint-disable-next-line no-await-in-loop
-				const usedSecrets = (await w.checkProofsSpent(t.proofs)).map(x => x.secret)
-				if (usedSecrets.length === t.proofs.length) {
-					// usedTokens.push(token)
-					continue
-				}
-				useableTokenProofs.push(...t.proofs.filter(x => !usedSecrets.includes(x.secret)))
+		const decoded = getDecodedToken(token)
```
Comment (BilligsterUser) [inline src/wallet/index.ts:60]: I dont understand what Changed Here?

File: src/components/nav/CustomDrawer.tsx [hunk 5]
```diff
@@ -4,8 +4,7 @@ import { ThemeContext } from '@src/context/Theme'
 import { highlight as hi, light } from '@styles'
 import { skipRoute } from '@util'
 import { useContext } from 'react'
-import { Image, StyleSheet, Text, View } from 'react-native'
-import { TouchableOpacity } from 'react-native-gesture-handler'
+import { Image, StyleSheet, Text, TouchableOpacity, View } from 'react-native'
 
 import { version } from '../../../package.json'
 // import { useState } from 'react'
```
Comment (BilligsterUser) [inline src/components/nav/CustomDrawer.tsx:10]: Why?

File: src/components/App.tsx [hunk 6]
```diff
@@ -16,14 +16,14 @@ import { KeyboardProvider } from '@src/context/Keyboard'
 import { ThemeContext } from '@src/context/Theme'
 import { addToHistory } from '@store/HistoryStore'
 import { dark, globals, light } from '@styles'
-import { formatInt, hasTrustedMint, isCashuToken, sleep } from '@util'
+import { formatInt, hasTrustedMint, isCashuToken, isErr, sleep } from '@util'
 import { initCrashReporting } from '@util/crashReporting'
 import { claimToken, isTokenSpendable, runRequestTokenLoop } from '@wallet'
```
Comment (BilligsterUser) [inline src/components/App.tsx:21]: Why and where so you Call this?

File: app.config.js [hunk 7]
```diff
@@ -1,6 +0,0 @@
-
```
Comment (BilligsterUser) [inline app.config.js:1]: Why did you delete this File?

**Review by BilligsterUser — CHANGES_REQUESTED** [review]
I have some questions @KKA11010

File: src/components/nav/CustomDrawer.tsx (hunk 5 above)
Comment (BilligsterUser) [reply src/components/nav/CustomDrawer.tsx:10]: So you need an commented import for that?

File: src/components/App.tsx (hunk 6 above)
Comment (KKA11010) [reply src/components/App.tsx:21]: on line 159 in the catch block:

async function initDB() {
			try {
				await initDb()
				runRequestTokenLoop()
				// await addAllMintIds()
				/* const test = await getProofs()
				l({ test }, await getBalance())
				l(await getBalancesByKeysetId()) */
			} catch (e) {
				l(e)
				alert(`Something went wrong while initializing the DB! ${isErr(e) ? e.message : ''}`)
			}
		}

File: app.config.js (hunk 7 above)
Comment (BilligsterUser) [reply app.config.js:1]: No i dont!
Removing this File Breaks our build !

File: src/components/nav/CustomDrawer.tsx (hunk 5 above)
Comment (BilligsterUser) [reply src/components/nav/CustomDrawer.tsx:10]: So the import is needed?
If not could you pls Remove it

File: app.config.js (hunk 7 above)
Comment (KKA11010) [reply app.config.js:1]: oh sorry. ok let me add it back

File: src/components/nav/CustomDrawer.tsx (hunk 5 above)
Comment (KKA11010) [reply src/components/nav/CustomDrawer.tsx:10]: sure i can remove it

**Comment by BilligsterUser** [comment]
Thx for the 2 left Things i need about 2 hours to Run some tests

**Review by BilligsterUser — CHANGES_REQUESTED** [review]

**Comment by KKA11010** [comment]
Fixes #55, Fixes #54

**Review by BilligsterUser — APPROVED** [review]
LGTM
